src/App.js

```diff
@@ -214,6 +256,17 @@ function App() {
             </Route>
             <Route path="/protest/:id">
               <ProtestPage />
+            <Route exact path="/sign-up">
+              <SignUp />
+            </Route>
+            <Route exact path="/leader-request">
+              <LeaderRequest user={state.user}/>
+            </Route>
+            <Route exact path="/leader-request-submitted">
+              <LeaderRequestSubmitted user={state.user}/>
+            </Route>
```

Comment:
Maybe this could be a nested route inside LeaderRequest? leader-request/submitted. And this route can live inside that file

Reply:
Moved to be a nested route 👍